### PR TITLE
Cleanup PostBind in coscheduling

### PR DESCRIPTION
**Pull request by czybjtu**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
this logic has been moved to PodGroup's controller.
#### Which issue(s) this PR fixes:

Fixes #https://github.com/kubernetes-sigs/scheduler-plugins/issues/531

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/controllers/podgroup_controller.go [hunk 1]
```diff
@@ -109,6 +111,11 @@ func (r *PodGroupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		if len(pods) >= int(pg.Spec.MinMember) {
 			pgCopy.Status.Phase = schedv1alpha1.PodGroupPreScheduling
 			fillOccupiedObj(pgCopy, &pods[0])
+		} else {
```
Comment (PiotrProkop) [inline pkg/controllers/podgroup_controller.go:114]: why add this else? Is this necessary to remove PostBind?

Comment (czybjtu) [reply pkg/controllers/podgroup_controller.go:114]: Yes, this else is unnecessary.
`PostBind` logic has been implmented in PodGroup controller. We can delete it as stated in comments.
https://github.com/kubernetes-sigs/scheduler-plugins/blob/2ea5a971705f0ba21052be7edff0f76323190ed7/pkg/coscheduling/core/core.go#L207-L209

Comment (PiotrProkop) [reply pkg/controllers/podgroup_controller.go]: 👍

File: pkg/controllers/podgroup_test.go [hunk 2]
```diff
@@ -109,14 +109,13 @@ func Test_Run(t *testing.T) {
 			podNextPhase:      v1.PodSucceeded,
 		},
 		{
-			name:              "Group status convert from scheduling to succeed",
+			name:              "Group status convert from pending to scheduling",
```
Comment (denkensk) [inline pkg/controllers/podgroup_test.go:112]: @czybjtu 
Can you explain why we need to change this test case here?

Comment (czybjtu) [reply pkg/controllers/podgroup_test.go:112]: This is a duplicate of line 102.
https://github.com/kubernetes-sigs/scheduler-plugins/blob/2ea5a971705f0ba21052be7edff0f76323190ed7/pkg/controllers/podgroup_test.go#L102

**Comment by denkensk** [comment]
/approve 

Thanks for your contribution. @czybjtu

**Comment by Huang-Wei** [comment]
@czybjtu could you please update the `release-notes` section of this issue? like:

```
Remove the PostBind implementation from the coscheduling plugin.
```

**Comment by czybjtu** [comment]
> @czybjtu could you please update the `release-notes` section of this issue? like:
> 
> ```
> Remove the PostBind implementation from the coscheduling plugin.
> ```

Done

**Pull request merged**